R/matlist.R

```diff
@@ -244,7 +244,6 @@ setMethod("zero_re", "mrgmod", function(.x, ...) {
 
 #' Methods for working with matrix-list objects
 #'
-#' @param .x a matlist object
```

Comment:
I didn't investigate the R devel change that started flagging this, but looks correct given that none of the (current) `*.matlist` methods have `.x` in their signatures.